clang-tools-extra/clang-tidy/misc/MisleadingBidirectionalCheck.h

```diff
@@ -13,6 +13,10 @@
 
 namespace clang::tidy::misc {
 
+/// Warn about unterminated bidirectional unicode sequence.
```

Comment:
```suggestion
/// Warns about unterminated bidirectional unicode sequence.
```
Need to adjust docs too